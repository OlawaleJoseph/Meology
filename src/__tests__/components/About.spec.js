import React from 'react';
import renderer from 'react-test-renderer';
import { Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import AboutPage from '../../pages/About';

describe('Home Page Snapshot', () => {
  it('builds home page correctly with search and meallist', () => {
    const tree = renderer.create(
      <Router history={createMemoryHistory()}>
        <AboutPage />
      </Router>,
    );
    expect(tree).toMatchSnapshot();
  });
});
