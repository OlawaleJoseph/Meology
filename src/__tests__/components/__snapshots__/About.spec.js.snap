// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Snapshot builds home page correctly with search and meallist 1`] = `
<section
  className="section about-section"
>
  <h1
    className="section-title"
  >
    about us
  </h1>
  <p>
    Hi, I'm Somoye Ayotunde. I'm a
     
    <strong>
      Software Engineer
    </strong>
     
    and I chose it because I found my true passion in programming. Whether figuring out a difficult problem or crafting beautiful user experience like
    <strong>
       MEOLOGY
    </strong>
    , programming satisfies my desire for both constant learning and ongoing refinement.
  </p>
  <p>
    <strong>
      MEOLOGY 
    </strong>
    retrieves data from
     
    <strong>
      The Meal DB
    </strong>
     
    API. You can get to view a comprehensive list of meals based on your searchTerm and can also view the full details of each meal by clicking on the details. It is built using
     
    <strong>
      React
    </strong>
     
    and also utilizes
     
    <strong>
      Redux
    </strong>
    .
  </p>
</section>
`;
